code/builders/builder-vite/src/codegen-set-addon-channel.ts

```diff
@@ -1,17 +1,14 @@
 export async function generateAddonSetupCode() {
   return `
     import { createChannel as createPostMessageChannel } from '@storybook/channel-postmessage';
-    import { createChannel as createWebSocketChannel } from '@storybook/channel-websocket';
     import { addons } from '@storybook/preview-api';
 
     const channel = createPostMessageChannel({ page: 'preview' });
     addons.setChannel(channel);
     window.__STORYBOOK_ADDONS_CHANNEL__ = channel;
     
     if (window.CONFIG_TYPE === 'DEVELOPMENT'){
-      const serverChannel = createWebSocketChannel({});
-      addons.setServerChannel(serverChannel);
-      window.__STORYBOOK_SERVER_CHANNEL__ = serverChannel;
+      window.__STORYBOOK_SERVER_CHANNEL__ = channel;
```

Comment:
It feels weird that both channels come from `'@storybook/channel-postmessage'`. Should instead we make a new `'@storybook/channel-duplex'` or something?

Reply:
Appreciate that. Maybe we should rename the package then (would that be bad too?) -- or at least indicate somewhere that 'channel-postmessage' is a bit of a misnomer as this now includes 2 channels.